components/camel-weather/src/main/java/org/apache/camel/component/weather/WeatherConfiguration.java

```diff
@@ -62,6 +65,14 @@
     private WeatherLanguage language = en;
     @UriParam
     private String headerName;
+    @UriParam
+    private String zip;
+    @UriParam
+    private List<String> ids;
+    @UriParam
+    private Integer cnt;
```

Comment:
Ah the weather api uses cnt instead. That is a bit of shame as cnt is a bad abbreviation. Also limit is a better english term to limit the result.